crates/bitwarden/src/vault/cipher/cipher.rs

```diff
@@ -77,6 +79,8 @@ pub struct CipherView {
     pub folder_id: Option<Uuid>,
     pub collection_ids: Vec<Uuid>,
 
+    pub key: Option<EncString>,
```

Comment:
Do we want to expose the key in a view?

Reply:
If we want to encrypt it back with the same Key, yeah. Though I guess we can generate a new key each time we encrypt a CipherView into a Cipher, if we wanted.

Reply:
Should we add a TODO or open a ticket for the future?